tests/robot-tests/README.md

```diff
@@ -27,17 +28,37 @@
 - [**Who should I talk to?**](#who-should-i-talk-to)
 
 
-## **What is this?**
+## What is this?
 This test framework runs UI tests against the Explore Education Statistics service using Selenium and Robot Framework.
 
-Currently, these tests are being maintained so they can be run on Windows. They're not being maintained for Linux or MacOS, but they may work without too much trouble. 
+Currently, these tests are being maintained so they can be run on Windows and Ubuntu
+## What do I need to install?
 
-## **What do I need to install?**
+Firstly, install Python 3.10. You can use pyenv to do this which is the recommended way of installing and managing python
+
```

Comment:
I'd be inclined to refer to pyenv's installation instructions here, rather than listing ways for each OS. Then they maintain the instructions for us.

pyenv install instructions: https://github.com/pyenv/pyenv#installation